testem.js

```diff
@@ -4,7 +4,6 @@ module.exports = {
   "disable_watching": true,
   "firefox_user_js": "./firefox_config.js",
   "launch_in_ci": [
-    "PhantomJS",
```

Comment:
I'd like to keep travis as much as possible. The main reason is that phantom is still used a lot by people that might use this addon. I'd rather skip this test in phantom (user-agent detection?) that remove it entirely.

Headless chrome can't come fast enough.